server/src/internalClusterTest/java/org/elasticsearch/indices/DataStreamIT.java

```diff
@@ -72,13 +77,19 @@
 
 public class DataStreamIT extends ESIntegTestCase {
 
+    @After
```

Comment:
This is fine, but I wonder if we should follow-up with generically removing all v2 templates like we do for regular templates?

Reply:
Yes, I noticed that component templates were not automatically removed after each test. I don't know why. @dakrone Is there a reason why component templates are automatically removed after each test?